model/Configuration.schema

```diff
@@ -0,0 +1,71 @@
+{
+    "type" : "object",
+    "properties" : {
+        "num_trajectories" : {"type" : "integer", "minimum": 0},
+        "population_path" : {"type" : "string"},
```

Comment:
I am not sure if this is included, but perhaps the string should be a path or at least non-empty string

Reply:
> I am not sure if this is included, but perhaps the string should be a path or at least non-empty string

Maybe empty string or string which ends with ".jld2"? Wouldn't empty string be useful if someone wanted to save a "template" configuration?

Reply:
OK, let's leave it as it is now